backoffice/backoffice/management/management/commands/delete_author.py

```diff
@@ -0,0 +1,64 @@
+from django.core.management.base import BaseCommand
+
+from backoffice.authors.models import AuthorWorkflow
+import uuid
+from django.conf import settings
+from opensearchpy import OpenSearch
+from opensearchpy.exceptions import NotFoundError
+
+
+class Command(BaseCommand):
+    """
+    A management command that delete a workflow by its UUID.
+    Removes it both from the database and from the OpenSearch.
+    >> python manage.py create_groups
+    """
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "uuid",  # positional argument, so it’s required
+            type=uuid.UUID,
+            help="A valid UUID (e.g., 123e4567-e89b-12d3-a456-426614174000)",
+        )
+
+    def handle(self, *args, **options):
+        connection = settings.OPENSEARCH_DSL["default"]["hosts"]
+        host, port = (
+            connection.split(":")
+            if isinstance(connection, str)
+            else connection[0].split(":")
+        )
+
+        client = OpenSearch(hosts=[{"host": host, "port": port}])
```

Comment:
We can create a utility function that will give you the opensearch_client as we will most probably re-use it in other clis that we will create. Something like
```
def get_opensearch_client():
    hosts_config = settings.OPENSEARCH_DSL.get("default", {}).get("hosts", [])
    ....
    return OpenSearch(hosts=hosts)

```

and then we call it like 
```
    client = get_opensearch_client()
```